src/schema/objects/metadata.yaml

```diff
@@ -2782,8 +2782,8 @@ RecordedEye:
   display_name: Recorded Eye
   description: |
     Indicates the eye tracked, for example, `"left"` or `"right"`.
-    It SHOULD be set to `"merged"` for recordings combining both eyes
-    typically generated by binocular eye-trackers.
+    It SHOULD be set to `"cyclopean"` for recordings combining both eyes
```

Comment:
Thanks! @mszinte I'll update the corresponding metadata to include this particular value and replace merged.

Reply:
Great

Reply:
Done